src/main/java/org/skriptlang/skript/bukkit/registration/BukkitSyntaxInfosImpl.java

```diff
@@ -37,7 +37,7 @@ static final class EventImpl<E extends SkriptEvent> implements Event<E> {
 
 		EventImpl(
 			SyntaxInfo<E> defaultInfo, ListeningBehavior listeningBehavior, String name,
-			@Nullable String since, @Nullable String documentationId, Collection<String> description, Collection<String> examples,
+			Collection<String> since, @Nullable String documentationId, Collection<String> description, Collection<String> examples,
```

Comment:
You need to update the since line to ensure its immutable:
`since = ImmutableList.copyOf(since)`